br/pkg/utils/backoff.go

```diff
@@ -194,11 +194,11 @@ func (bo *pdReqBackoffer) NextBackoff(err error) time.Duration {
 	// bo.attempt--
 	e := errors.Cause(err)
 	switch e { // nolint:errorlint
-	case nil, context.Canceled, context.DeadlineExceeded, io.EOF, sql.ErrNoRows:
+	case nil, context.Canceled, context.DeadlineExceeded, sql.ErrNoRows:
```

Comment:
I think we'd better to figure out why `io.EOF` isn't retryable from beginning.

Reply:
I guess just move code from lightning.